contributing.md

```diff
@@ -0,0 +1,15 @@
+# Contributing to FingerprintJS Pro Flutter integration
+
+## Development Environment
+
+Before starting work on the repository please configure the environment and emulators.
+1. Install Flutter using [instruction for your OS](https://docs.flutter.dev/get-started/install).
+2. Follow official recommendations [about setting up IDE](https://docs.flutter.dev/get-started/editor?tab=androidstudio).
+
+## Development playground
```

Comment:
Maybe we should also mention that we use an approach with PRs? Something like this:

> The main branch is locked for the push action. For proposing changes, use the standard [pull request approach](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request). It's recommended to discuss fixes or new functionality in the Issues, first.

Reply:
Thank you for the proposal, added!